client/build-config/src/paths.ts

```diff
@@ -19,15 +20,38 @@ export function resolveWithSymlink(...args: string[]): string {
     }
 }
 
+export function resolveAssetsPath(root: string): string {
+    if (IS_BAZEL && process.env.WEB_BUNDLE_PATH) {
+        return resolveWithSymlink(
+            root,
+            process.env.WEB_BUNDLE_PATH
+        )
+    }
+
+    if (process.env.NODE_ENV && process.env.NODE_ENV == "development") {
+        return resolveWithSymlink(root, 'ui/assets')
+    }
+
+    // With Bazel we changed how assets gets bundled. Previsouly, we would just put the assets at /ui/assets/.assets
+    // and be done with it. With Bazel, we have different loaders on the backend where the assets gets embedded. So
+    // what we do here is "simulate" what happens in bazel, by putting the assets in the correct relative directory
+    // so that when the backend is compiled the assets gets embedded properly
+    let isEnterprise: boolean = getEnvironmentBoolean("ENTERPRISE");
+    let relativeAssetPath: string = isEnterprise ? "enterprise" : "oss"
```

Comment:
Types are automatically inferred by Typescript here, so we can safely omit them. And the `let` –> `const` is here to make it consistent with other variable declarations. We use `let` when we plan to reassign the variable later to a different value, otherwise, `const` is used.

```suggestion
    const isEnterprise = getEnvironmentBoolean("ENTERPRISE");
    const relativeAssetPath = isEnterprise ? "enterprise" : "oss"
```